smoke-testing/src/test/java/games/strategy/engine/data/GameTestUtils.java

```diff
@@ -73,7 +75,7 @@ public static ServerGame setUpGameWithAis(GameSelectorModel gameSelector) {
       playerTypes.put(player.getName(), PlayerTypes.PRO_AI);
     }
     Set<Player> gamePlayers = gameData.getGameLoader().newPlayers(playerTypes);
-    HeadlessLaunchAction launchAction = new HeadlessLaunchAction();
+    HeadlessLaunchAction launchAction = new HeadlessLaunchAction(any());
```

Comment:
How does this work - this isn't a mock expectation... ?

Reply:
Actually my bad. It works because any() just returns null which is fine in this case. But yeah, it really should be a mock